app/src/main/java/com/tasomaniac/openwith/settings/SettingsFragment.java

```diff
@@ -226,11 +227,10 @@ private void displayLicensesDialogFragment() {
     }
 
     private void startContactEmailChooser() {
-        ShareCompat.IntentBuilder.from(getActivity())
-                .addEmailTo("Said Tahsin Dane <tasomaniac+openlinkwith@gmail.com>")
-                .setSubject(getString(R.string.app_name))
-                .setType("message/rfc822")
-                .startChooser();
+        EmailIntentBuilder.from(getActivity())
+                .to("tasomaniac+openlinkwith@gmail.com")
```

Comment:
`EmailIntentBuilder` tries hard to only create standard-compliant URIs because I believe formal standards are very important for interoperability. If you use something outside of the standard that happens to work with the most important implementations you contribute to making this thing an informal standard. If independent implementations don't want to be left behind they have to (know of and) play by the unofficial rules. Rules that aren't written down anywhere. The easiest way to avoid this situation is to encourage everyone to play by the official rules. If necessary you can change those rules through the proper channels.